docs: "What's new in Svelte" August newsletter (#1433)
sveltejs/svelte.dev: apps/svelte.dev/content/blog/2025-08-01-whats-new-in-svelte-august-2025.md

---
title: "What's new in Svelte: August 2025"
description: 'Async Svelte is out - with all the fixins'
author: Dani Sandoval
authorURL: https://dreamindani.com
---

> With [Async Svelte](https://github.com/sveltejs/svelte/discussions/15845) officially out in the world and its companion [Remote Functions](https://github.com/sveltejs/kit/discussions/13897) already ligned up, there's a lot of updates to share!

Reviewer: Rich just merged and released remote functions 😄 https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%402.27.0
```suggestion
With [Async Svelte](https://github.com/sveltejs/svelte/discussions/15845) and its companion [Remote Functions](https://github.com/sveltejs/kit/discussions/13897) officially out in the world, there's a lot of updates to share!
```

> Be sure to check out the showcase for videos from X, Y and Z about all the brand new features in Svelte.

Reviewer: Are X, Y, and Z meant to be placeholders?
```suggestion
Be sure to check out the showcase for videos from Rich, Paolo, and Stanislav about all the brand new features in Svelte.
```

Let's dive into the highlights...

## What's new in Svelte, SvelteKit, Language Tools and CLI (sv)

- `getAbortSignal` returns an `AbortSignal` that aborts when the current derived or effect re-runs or is destroyed (**svelte@5.35.0**, [Docs](https://svelte.dev/docs/svelte/svelte#getAbortSignal), [#16266](https://github.com/sveltejs/svelte/pull/16266))

> - A new `parent` property has been added to the `__svelte_meta` properties that are added to elements at dev time - useful for debugging nested hierarchies (**svelte@5.35.1**, Docs, [#16255](https://github.com/sveltejs/svelte/pull/16255))

Reviewer: ```suggestion
- A new `parent` property has been added to the `__svelte_meta` properties that are added to elements at dev time - useful for debugging nested hierarchies (**svelte@5.35.1**, [#16255](https://github.com/sveltejs/svelte/pull/16255))
```

> - `await` is now supported in components when using the `experimental.async` compiler option (**svelte@5.36.0/extensions@109.10.0**, [Docs](https://svelte.dev/docs/svelte/await-expressions) [#15844](https://github.com/sveltejs/svelte/pull/15844))

Reviewer: Remote functions was just released in https://github.com/sveltejs/kit/releases/tag/%40sveltejs%2Fkit%402.27.0

Docs: https://svelte.dev/docs/kit/remote-functions
PR: https://github.com/sveltejs/kit/pull/13986

> - `await` is now supported in components when using the `experimental.async` compiler option (**svelte@5.36.0/extensions@109.10.0**, [Docs](https://svelte.dev/docs/svelte/await-expressions) [#15844](https://github.com/sveltejs/svelte/pull/15844))

Reviewer: ```suggestion
- `await` is now supported in components when using the `experimental.async` compiler option (**svelte@5.36.0/extensions@109.10.0**, [Docs](https://svelte.dev/docs/svelte/await-expressions), [#15844](https://github.com/sveltejs/svelte/pull/15844))
```

description: 'Async Svelte is out - with all the fixins'
author: Dani Sandoval
authorURL: https://dreamindani.com
---

With [Async Svelte](https://github.com/sveltejs/svelte/discussions/15845) officially out in the world, there's a lot of updates to share!

Be sure to check out the showcase for videos from X, Y and Z about all the brand new features in Svelte.

Let's dive into the highlights...

## What's new in Svelte, SvelteKit, Language Tools and CLI (sv)

- getAbortSignal returns an AbortSignal that aborts when the current derived or effect re-runs or is destroyed (**svelte@5.35.0**, [Docs](https://svelte.dev/docs/svelte/svelte#getAbortSignal), [#16266](https://github.com/sveltejs/svelte/pull/16266))
- A new parent property has been added to the `__svelte_meta` properties that are added to elements at dev time - useful for debugging nested hierarchies (**svelte@5.35.1**, Docs, [#16255](https://github.com/sveltejs/svelte/pull/16255))
- `await` is now supported in components when using the `experimental.async` compiler option (**svelte@5.36.0/extensions@109.10.0**, [Docs](https://svelte.dev/docs/svelte/await-expressions) [#15844](https://github.com/sveltejs/svelte/pull/15844))

> - `svelte.config.ts` can now be loaded in runtimes that support importing TypeScript files (**kit@2.23.0**, [#13935](https://github.com/sveltejs/kit/pull/13935))

Reviewer 2: this does not yet work with language-tools and eslint-plugin-svelte, i'd prefer it to be announced more publicly after all our tools support it
```suggestion

```

> - There are now types for the `params` prop in page/layout components (**kit@2.24.0/extensions@109.10.0**, [#13999](https://github.com/sveltejs/kit/pull/13999))

Reviewer: Similarly, typed routes are a thing now https://github.com/sveltejs/kit/pull/13864
